letsencrypt-auto

```diff
@@ -3,16 +3,29 @@
 # A script to run the latest release version of the Let's Encrypt in a
 # virtual environment
 #
+# ./letsencrypt-auto     --      run the latest public release from PyPI
+#
+# ./lesencrypt-dev       --      run the copy from the local tree, including
+#                                experimental plugins
+#
 # Installs and updates the letencrypt virtualenv, and runs letsencrypt
 # using that virtual environment.  This allows the client to function decently
 # without requiring specific versions of its dependencies from the operating
 # system.
 
 XDG_DATA_HOME=${XDG_DATA_HOME:-~/.local/share}
-VENV_NAME="letsencrypt"
+if [ "`basename \"$0\"`" = "letsencrypt-dev" ] ; then
```

Comment:
Yes, forgot to git add the link.  I'm fine factoring this out into another script, though...